signOut.js

```diff
@@ -4,7 +4,7 @@ import { ActionOptions, SignOutUserActionContext } from "gadget-server";
  * @param { SignOutUserActionContext } context
  */
 export async function run({ params, record, logger, api, session }) {
-  // unset the associated user on the active session
+  // Unsets the signed-in user from the active session
```

Comment:
```suggestion
  // Removes the signed-in user from the active session
```

Reply:
Another suggestion: Is it confusing to say 'signed-in user' here? I don't have strong feelings either way
```suggestion
  // Removes the user from the active session
```